src/components/Ublock/Ublock.tsx

```diff
@@ -16,6 +16,7 @@ export interface UblockProps {
     noBlock?: boolean
     wide?: boolean
     noPadding?: boolean
+    className?: string | undefined
```

Comment:
Is there a need for undefined as you have question mark there?

Reply:
removed undefined